nucypher/blockchain/eth/deployers.py

```diff
@@ -310,7 +284,7 @@ def __init__(self, secret_hash, *args, **kwargs):
         self.secret_hash = secret_hash
 
     def deploy(self) -> Dict[str, str]:
-        self.ready_to_deploy
+        self.check_deployment_readiness
```

Comment:
Why is this not being called?  (maybe fixed in a later commit)

Reply:
Good Catch!